README.md

```diff
@@ -60,7 +60,7 @@ Planck.js is [available on jsDelivr](https://www.jsdelivr.com/projects/planck).
 ## Usage
 
 Planck.js does not use any renderer by default. To use or integrate it with a rendering library all you need to do 
-is calling `world.step(1/hz)` in each frame, and then iterating over world entities to draw or update them.
+is call `world.step(1/hz)` in each frame, and then iterate over world entities to draw or update them.
```

Comment:
I assume you mean noun? In that case then calling is correct, though I admit then I struggle to understand its usage here.